src/main/java/org/walkmod/OptionsBuilder.java

```diff
@@ -30,26 +36,49 @@
  * @author abelsromero
  */
 public class OptionsBuilder {
+	private static final String DEFAULT_PATH = ".";
 
 	/**
-	 * Options instance
+	 * Collected options
 	 */
-	private Options options;
+	private final Map<String, Object> options = new HashMap<String, Object>();
```

Comment:
There are several questions here:

- If we can avoid imports in OptionBuilder that already appear in Option, please move them.

- You are right that public constants, can be replaced by private 👍 

-  I do not have an strong opinion about using nulls. It is just what I used to do, so if you think that using empty is better, please proceed. Could it be that empty cannot be modified? So, probably, in that case, if we are using immutable values, it makes sense.

- Yes, please, mark them as deprecated. These are used by the Maven and Gradle plugins that run WalkMod.